proposal/pachyderm.md

```diff
@@ -0,0 +1,246 @@
+
+
+**Authentication**
+Community-Edition Pachyderm does not provide authentication features. However, Enterprise-Edition Pachyderm does provide authentication features and the possibility to plug into auth services such as Keycloak.
+
+**Endpoint**
+Pachyderm cluster can be accessiable via pachd gGRP port 1650. In OpenShift, end-users can use internal service hostname or External IP(If loadbalancer support HTTP2. OSD on AWS work)/port-forward way.
+
+**How to configure your Notebook to connect to your pachyderm cluster**
+
+  - Before experimenting with Pachyderm in notebook, run the following command line in a new cell:
+
+      The command installs Pachyderm’s CLI (`pachchtl`) in your notebook environment.
+
+      **Note**: The version of `pachctl` must match the version of Pachyderm deployed by the Operator. Pachyderm’s version is available on the Operator’s description.
+
+
+      At a minimum, you should always use the identical major & minor versions of `pachctl` and Pachyderm (`pachd`). Note that the following package `/v2.0.2/pachctl_2.0.2_linux_amd64.tar.gz` (see command line below) refers to version 2.0.2 of Pachyderm. Depending on the version currently deployed by the Operator, you might need to update the version of `pachctl` (i.e.change `2.0.2` to `2.1.0` for example) in the command line.
+
+      ```shell
+      ! curl -o /tmp/pachctl.tar.gz -L https://github.com/pachyderm/pachyderm/releases/download/v2.0.2/pachctl_2.0.2_linux_amd64.tar.gz && tar -xvf /tmp/pachctl.tar.gz  -C /tmp && cp /tmp/pachctl_2.0.2_linux_amd64/pachctl  /opt/app-root/bin/
+      ```
+
+  - Create a new context for your CLI to connect to your cluster:
+
+      In the following command, update **“pachyderm-operator-install”** in the pachd_address with your project name. The pattern should be: `pachd.`your-project-name`.svc.cluster.local:30650`.
+
+      ```shell
+      !echo '{"pachd_address":"pachd.pachyderm-operator-install.svc.cluster.local:30650"}' | pachctl config set context pachyderm --overwrite
+      ```
+
+  - Finally, switch `pachctl` context to your newly created context:
+      ```shell
+      !pachctl config set active-context pachyderm
+      ```
+
+  - Verify that your active context is the new context pachyderm
+      ```shell
+      !pachctl config get active-context
+      ```
+
+  - You are all set. Run: 
+      ```shell
+      !pachctl version 
+      ```
+      This should print the versions of pachctl and Pachyderm deployed. 
+      For example:
+      ```
+      COMPONENT           VERSION             
+      pachctl             2.0.2             
+      pachd               2.0.2
+      ```
+
+You are ready to run your first pipelines in a notebook. Check our **Quick Start** in the `Enabled` Menu of your OpenShift Data Science Console. You can later extend this first notebook by following the steps in this written [tutorial](https://docs.pachyderm.com/latest/getting_started/beginner_tutorial/) on our documentation website.
+
```

Comment:
What is "OpenShift Data Science Console"

Reply:
I will delete the part because it explains RHODS based.

Reply:
Updated